packages/nuxi/src/commands/analyze.ts

```diff
@@ -11,7 +11,7 @@ import { defineNuxtCommand } from './index'
 export default defineNuxtCommand({
   meta: {
     name: 'analyze',
-    usage: 'npx nuxi analyze [rootDir]',
+    usage: 'npx nuxi analyze [--log-level, -l] [rootDir]',
```

Comment:
On casing, we probably support case translation from citty, but i probably choose normalization to camel case matching JS programmatic API when calling it. Also on shortcut it could be useful to reserve it for future purposes such as `--listen, -l` in some subdommands.

Reply:
Existing options are all lower case and in some cases kebab case (e.g. `ssl-key`, `ssl-cert`). I think we should keep this kebab-case for consistency until `citty` supports camel-case normalisation.

Do you think we should require long form `--log-level` to avoid shadowing potential `--listen` shortcut? Very happy to remove `args.l` support...

Reply:
For mapping, i meant mapping to nuxt config btw. But still should be supported with both conventions so that's okay.

Would be nice to remove shortcut indeed 👍🏼 (we could introduce `--verbose` / `--silent` later since are kinda de-facto convention for CLI logging controls)